doc/theoretical-max-tps.md

```diff
@@ -24,16 +26,26 @@ Finally, don't forget to shut down your EKS cluster.
-| 22.2.0 | `stellar/unsafe-stellar-core:22.2.0-2361.e6c1f3bfc.focal-perftests` | `b49c0810f159e0305328e127057e1f6fc08a0524` | BucketListDB | 7 / 3 | m5d.4xlarge | 10 | 1079 | Performance improvement due to BucketList caching changes |
-| 22.1.0rc1 | `stellar/unsafe-stellar-core:22.1.0-2189.rc1.fdd833d57.focal-perftests` | | BucketListDB | 7 / 3 | m5d.4xlarge | 10 | 989 | Performance improvement due to [networking changes](https://github.com/stellar/stellar-core/pull/4544) |
-| 22.0.0 | `stellar/unsafe-stellar-core:22.0.0-2138.721fd0a65.focal-perftests` | | BucketListDB | 7 / 3 | m5d.4xlarge | 10 | 902 | |
-| 22.0.0rc2 | `stellar/unsafe-stellar-core:22.0.0-2095.rc2.1bccbc921.focal-perftests` | | BucketListDB | 7 / 3 | m5d.4xlarge | 10 | 958 | First version with mandatory BucketListDB backend |
-| 21.3.1 | `stellar/unsafe-stellar-core:21.3.1-2007.4ede19620.focal-perftests` | | BucketListDB | 7 / 3 | m5d.4xlarge | 10 | 1110 | |
-| 21.3.1 | `stellar/unsafe-stellar-core:21.3.1-2007.4ede19620.focal-perftests` | | SQLite in-memory | 7 / 3 | m5d.4xlarge | 10 | 1170 | |
-| 21.2.0 | `stellar/unsafe-stellar-core:21.2.0-1953.d78f48eac.focal-perftests` | | BucketListDB | 7 / 3 | m5d.4xlarge | 10 | 1059 | |
-| 21.2.0 | `stellar/unsafe-stellar-core:21.2.0-1953.d78f48eac.focal-perftests` | | SQLite in-memory | 7 / 3 | m5d.4xlarge | 10 | 1053 | |
-| 21.1.0 | `stellar/unsafe-stellar-core:21.0.1-1917.52a449ff3.focal-testing-asan-disabled-perftests` | | SQLite in-memory | 7 / 3 | m5d.4xlarge | 10 | 1137 | |
+stellar-core release. The columns are as follows:
+
+* **Core Version.** The stellar core release tested.
+* **Core Image.** The exact docker image tested, from Docker Hub.
+* **Core Compiler Flags.** C/C++ compiler flags used in the stellar-core build.
+* **Core Configure Flags.** Arguments passed to stellar-core's `./configure` script.
+* **Supercluster Commit.** Supercluster commit hash used to run the test
+* **Extra Supercluster Options.** Additional Supercluster commands used beyond the ones specified in the template specified in the introduction of this document.
+* **Max TPS.** The result of the test.
+* **Notes.** Any additional information on the test result or methodology.
+
+| Core Version | Core Image | Core Compiler Flags | Core Configure Flags | Supercluster Commit | Extra Supercluster Options | Max TPS | Notes |
+|--------------|------------|---------------------|----------------------|---------------------|----------------------------|---------|-------|
+| 22.3.0 | `stellar/unsafe-stellar-core:22.3.1-2490.e643061a4.focal-tmtps-perftests` | `-ggdb -O3 -fstack-protector-strong` | | `d9df3be5ec3ea6d7f18262c601b05c793eb2c63b` | `--enable-relaxed-auto-qset-config` | 2032 | Performance improvement due to a combination of new stellar-core performance focused features, and testing methodology changes due to pregenerating transactions for load generation. See [release notes](https://github.com/stellar/stellar-core/releases/tag/v22.3.0) for more info. |
```

Comment:
I think we should add `--run-for-max-tps` as well since previous results didn't use it.

Reply:
Done!